src/test/resources/saml2.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
are there any security concerns by publishing this valid assertion?

Reply:
I think not, because I hope that this assertion is valid only for DEV purposes

Reply:
it has been already committed, you have to squash the commits to be sure that this won't appear on the history